### DISCUSSION TITLE
Unexpected import results

### DISCUSSION THREAD

**Discussion by MarianiVentura** [dfinke/ImportExcel#1291] [Q&A] [answered]
Good day.

I have just started using Powershell and the import-excel module.  So far, it has been awesome -- thank you for this wonderful tool.

I have a situation that I cannot understand.  I have a very simple xlsx file I am trying to import, but only some of the data is importing.
Here is my simple file:
![image](https://user-images.githubusercontent.com/21973914/197358036-d8d63235-a16c-4ab6-9a30-707f1012c991.png)

It has 27 rows, including the header.  No matter what I try, I cannot get columns 3 thru 6 to populate on import.  Also, I get 55 rows in my object count?

![image](https://user-images.githubusercontent.com/21973914/197358104-7168cd94-a063-4fad-88dd-7189d329c48a.png)

![image](https://user-images.githubusercontent.com/21973914/197358127-c508b947-54eb-4cbc-8cdc-869f3c6a2af7.png)

I am pretty sure this is a newbie error, but I am really struggling to figure out.  

I appreciate any advice on this issue.  Thank you again for making such a wonderful tool.

**Comment by dfinke** [comment]
Please post the xlsx here

**Reply by MarianiVentura** [reply]
[order.xlsx](https://github.com/dfinke/ImportExcel/files/9845219/order.xlsx)

Thank you!

Michael

**Reply by stahler** [reply, marked as answer]
Your workbook has two worksheets.
It is pulling the first one correctly dms_po_l.  If you want to import the second, you can specify the worksheetname.

```powershell
Import-Excel '.\order.xlsx' -WorksheetName Example
```

**Comment by MarianiVentura** [comment]
Aha!  As I mentioned, I am a newbie and that is definitely a rookie error.  Thank you both so much for taking the time to respond and so promptly.  

It is very much appreciated.

**Comment by dfinke** [comment]
@MarianiVentura It's always ok to ask. This also leaves breadcrumbs for other folks new to the module who can hit same issue.
